Catappult/appcoins-wallet-android
# [feature/APPC-2561] New My Wallets

**What does this PR do?**

   Implements the new My Wallets screen fit for the new bottom navigation.

**Database changed?**

   No

**How should this be manually tested?**

  Check every flow for the new My Wallets and check that it is working as intended.

**What are the relevant tickets?**

  [APPC-2561](https://aptoide.atlassian.net/browse/APPC-2561)


**Code Review Checklist**
